cmd/rule-evaluator/main.go

```diff
@@ -560,6 +570,8 @@ func sendAlerts(s *notifier.Manager, externalURL *url.URL) rules.NotifyFunc {
 			}
 			if externalURL != nil {
```

Comment:
suggestion: Change the condition to check if the `externalURL` is prefixed with the (suggested) GCM `const`.

e.g.
```go
// If it's a GCM link (default), create the full URL for the alert.
if strings.HasPrefix(externalURL, gcmURL) {
	a.GeneratorURL = googleCloudLink(externalURL, expr, alert.FiredAt, alert.FiredAt.Add(-time.Hour))
// Otherwise, if it was specified we assume it points to a Prometheus frontend.
} else if externalURL != nil {
	a.GeneratorURL = externalURL.String() + strutil.TableLinkForExpression(expr)
}
```

Reply:
I'm not sure if prefix makes sense. I made it check for equals, which would be the normalized URL since it's already been parsed. We can always change it to prefix if/when we add full templating support.

Reply:
Ah - agreed, this is better. Thanks!